### PR TITLE
Add new local strategy

### PR DESCRIPTION
**Pull request by augusto-jm-amaral**
- Add parameter `local` to use in local tests.
- Fix screenshot folder path.
- Update README (Add support list and update usage).
- Update examples (Add mobile test example).
  
Resolves: #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,17 @@
 
 Robrowser is a bot to help you test your project in many different browsers and operating systems.
 
-It started as a automation for [Browser Stack](https://www.browserstack.com) and its goal is to help you validate how your project look in many different environments by taking screenshots for you.
+## Support
 
-[Saucelabs](https://saucelabs.com/) is also currently accepted and if you would like to see other cross-browser tester running open an issue or a pull-request so we can check it!
+The supported test hubs are:
```
Comment (vcapretz) [inline README.md]: `The currently supported...`

File: README.md [hunk 2]
```diff
@@ -3,9 +3,17 @@
...
+
+- [Browser Stack](https://www.browserstack.com)
+
+We're adding support for other hubs, to be added.
```
Comment (vcapretz) [inline README.md]: `The next hubs we intend to support are:`

maybe here you can link the issue number so people can keep track of it

File: index.js [hunk 3]
```diff
@@ -22,20 +26,43 @@ const loadJSON = (filePath) => {
...
-  spinner.text = getEndTestMessage(finishedTestsCount, total, finalConcurrency)
+  spinner.text = getStatusMessage(finishedTestsCount, total, finalConcurrency)
 }
 
 const endAllTests = () => {
   spinner.stop()
 }
 
+const stopBrowserstackLocal = () => {
+  spinner.text = 'Stop browserstack tunnel'
```
Comment (vcapretz) [inline index.js]: `Stoping`

**Comment by vcapretz** [comment]
nice! this is really a great solution, instead of the old global local 🎉

File: README.md (hunk 1 above)
Comment (augusto-jm-amaral) [reply README.md]: nice, I'm going to adjust.

**Comment by augusto-jm-amaral** [comment]
I already finished the corrections, @vcapretz.

**Pull request merged**